src/app/components/shell/sidebar/library/AppShellSidebarLibrary.vue

```diff
@@ -0,0 +1,200 @@
+					</span>
+					<span class="shell-nav-label">
+						<AppTranslate>Followed Games</AppTranslate>
+					</span>
+				</RouterLink>
+			</li>
+
+			<li v-if="ownedCollection && !Screen.isXs" class="offline-disable">
+				<RouterLink
+					v-app-track-event="`sidebar:collection:owned`"
+					:to="ownedCollection.routeLocation"
+					active-class="active"
+				>
+					<span class="shell-nav-icon">
+						<AppJolticon icon="heart-filled" />
+					</span>
+					<span class="shell-nav-label">
+						<AppTranslate>Owned Games</AppTranslate>
+					</span>
+				</RouterLink>
+			</li>
+		</ul>
+
+		<template v-if="user">
+			<!-- Playlists -->
+			<template v-if="isLibraryBootstrapped">
+				<div class="nav-heading" :title="$gettext(`Playlists`)">
+					<AppTranslate>Playlists</AppTranslate>
+					<span class="badge">
+						{{ collectionsLength }}
+					</span>
+				</div>
+
+				<div class="nav-controls">
+					<AppButton
+						v-app-track-event="`sidebar:playlist-add`"
+						primary
+						block
+						class="offline-disable"
+						@click="showAddPlaylistModal"
+					>
+						<AppTranslate>New Playlist</AppTranslate>
+					</AppButton>
+				</div>
+
+				<div v-if="collections.length > 0" class="nav-controls">
+					<input
+						v-model="playlistFilterQuery"
+						type="search"
+						class="form-control"
+						:placeholder="$gettext(`Filter playlists...`)"
+					/>
+				</div>
+				<div v-else class="-no-playlists alert">
+					<p>
+						<AppTranslate>
+							Create playlists to organize and share the games in your library.
+						</AppTranslate>
+					</p>
+					<p>
+						<AppTranslate>
+							Follow other people's playlists to discover more games!
+						</AppTranslate>
+					</p>
+				</div>
+
+				<!-- Playlist Folders -->
+				<div v-for="key of playlistFoldersToDisplay" :key="key">
+					<ul class="shell-nav">
+						<li class="offline-disable">
+							<a @click="toggleFolder(key)">
+								<span class="shell-nav-icon">
+									<AppJolticon
+										:icon="
+											openFolders.indexOf(key) === -1
+												? 'folder'
+												: 'folder-open'
+										"
+									/>
+								</span>
+								{{ playlistFolders[key].title }}
+							</a>
+							<AppExpand :when="openFolders.indexOf(key) !== -1">
+								<AppShellSidebarCollectionList
+									:collections="playlistFolders[key].collections.value"
+									:filter="playlistFilterQuery"
+									:should-sort="key === 'developers'"
+								/>
+							</AppExpand>
+						</li>
+					</ul>
+				</div>
+
+				<!-- Main Playlists (not in folders) -->
+				<AppShellSidebarCollectionList
+					:collections="mainPlaylists"
+					:filter="playlistFilterQuery"
+				/>
+			</template>
+		</template>
+	</div>
+</template>
+
+<style lang="stylus" scoped>
+#shell-sidebar-library
+	padding-top: 16px
+
+	.-no-playlists
+		margin-left: ($grid-gutter-width-xs / 2)
+		margin-right: ($grid-gutter-width-xs / 2)
+		background-color: var(--theme-bg-subtle)
```

Comment:
does this need to be nested?

Reply:
i think this was an old style so that we didn't have conflicts in the CSS definitions